i18n/zh-CN/docusaurus-plugin-content-docs/current/ts-graphviz/11-advanced-topics/02-package-architecture.md

```diff
@@ -0,0 +1,60 @@
+---
+description: ts-graphviz 的包架构。
+---
+# 包架构
+
+**ts-graphviz** 由多个包组成，每个包具有特定的功能：
+
+- **`ts-graphviz`**（主包）：提供用于创建和操作图表的高级 API，适合大多数用户。
+
+- **`@ts-graphviz/core`**：包含模型和函数的核心实现，供内部使用并可供高级用户使用。
+
+- **`@ts-graphviz/common`**：聚合 Graphviz 领域知识，提供类型定义、常量和实用工具。支持扩展自定义类型和属性等用例。
```

Comment:
The description is correct because it does not mean that it is not publicly available, but only that it is not available as a submodule of ts-graphviz.